README.md

```diff
@@ -1,46 +1,49 @@
 # iris-project
 
-## How to update the site
+## How do I update the site?
 
 Updating is just a matter of commiting and pushing your changes. Github-pages will automatically observe the change and in a matter of seconds the main home-page will be updated. 
 
-## How to add publications
+## How do I test the site locally?
 
-To add a new publication find
-```html
-    <dl class="ref">
+Assuming you have a working and decently recent Ruby installation with `bundler` (`gem install bundler`) you should be able to set everything up by running at the root of the repository to install all dependencies (including Jekyll).
+```bash
+bundle install
```

Comment:
When I do `bundle install`, it says
```
Your user account isn't allowed to install to the system RubyGems.
  You can cancel this installation and run:

      bundle config set --local path 'vendor/bundle'
      bundle install

  to install the gems into ./vendor/bundle/, or you can enter your password
  and install the bundled gems to RubyGems using sudo.
```
I certainly won't give this thing root rights... is there a way to have "local switches" or whatever they call them?

Reply:
I guess the instructions are telling me what to do. ;) My point is, we should adjust the README as well. Recommending to install things system-wide is bad advice IMO.

Reply:
I ended up using
```
bundle config set --local path '.bundle'
```
to match the `.bundle` in the `.gitignore`. Not sure if that's what you intended with that ignore line?

Reply:
Yeah, I did run the same command. And yes, that is what I intended. :)

Reply:
Done.